README.md

```diff
@@ -72,28 +80,90 @@ This template follows a very simple project structure:
 
 #### Android
+
 A map associating builds with env files is already defined in `app/build.gradle` you can modify or add environments if needed.
 
 For multiple enviroments to work you would need to change `-keep class [YOUR_PACKAGE_NAME].BuildConfig { *; }` on `proguard-rules.pro` file.
 
 #### iOS
+
 The basic idea in iOS is to have one scheme per environment file, so you can easily alternate between them.
 
 To create a new scheme:
+
 - In the Xcode menu, go to Product > Scheme > Edit Scheme
 - Click Duplicate Scheme on the bottom
 - Give it a proper name on the top left. For instance: "qa"
 - Then edit the newly created scheme to make it use a different env file. From the same "manage scheme" window:
 
 Expand the "Build" settings on left
+
 - Click "Pre-actions", and under the plus sign select "New Run Script Action"
 - Where it says "Type a script or drag a script file", type: `echo ".env.qa" > /tmp/envfile` replacing `.env.qa` with your file.
 
 ## Styleguide
+
 For coding styling we decided to go with ESLint and [React Native community's styleguide](https://github.com/facebook/react-native/tree/master/packages/eslint-config-react-native-community#readme).
+
+# How to use it
+
+The idea of this section is to explain how the template composition is better and easier to use when you try to use well formed arquitectures, especially using redux flow.
+
+First keep in mind that you will see the same way of classification in multiple places of the app. Different resources orchestrated by an index.js that allow you to call it as a single instance. This structure lets you use a simple way to import things.
+
+With that in mind we are going to look at each folder to explain how to use it.
+
+## Components
+
+The components are the basic structure of an application in react native, but specifically for this template the idea of ​​keeping the complexity of the development to a minimum, for this the ideal is that all the components are at the same nesting folder level. A component is only created if a specific view or functionality is being worked on or if the same object is to be reused in multiple parts of the application.
+
+Another important thing is the use of propTypes to check the kind of data that your components need to work properly. If the component receives some data from others, the type of this props must be defined and in case you need it the default value of the property too.
+
+### Static resources:
+
+To keep an application scalable and organized, the global static resources that are used in the application have to be created in a specific file.
+
+### We manage three main folders for that:
+
+- Assets: Here you can store all the images and icons that you need through the app. You have as example the icon ic_home.png, to respond with the different device screen densities just create inside the same folder the image and all the scaled versions that you need.
```

Comment:
I may be wrong but I think RN only handles x1 x2 and x3, so maybe we should advise to use this scales